sycl/source/detail/allowlist.cpp

```diff
@@ -241,6 +254,27 @@ AllowListParsedT parseAllowList(const std::string &AllowListRaw) {
     }
   }
 
+  if (IsDeprecatedKeyNameDeviceNameWasUsed &&
+      IsDeprecatedKeyNamePlatformNameWasUsed) {
+    std::cerr << "\nWARNING: " << DeprecatedKeyNameDeviceName << " and "
+              << DeprecatedKeyNamePlatformName
+              << " in SYCL_DEVICE_ALLOWLIST are deprecated. ";
+  } else if (IsDeprecatedKeyNameDeviceNameWasUsed) {
+    std::cerr << "\nWARNING: " << DeprecatedKeyNameDeviceName
+              << " in SYCL_DEVICE_ALLOWLIST is deprecated. ";
+  } else if (IsDeprecatedKeyNamePlatformNameWasUsed) {
+    std::cerr << "\nWARNING: " << DeprecatedKeyNamePlatformName
```

Comment:
Are you sure `std::cerr` is the right stream for a warning message? `std::cerr` is intended for error output.

Reply:
Sounds reasonable, fixed in https://github.com/intel/llvm/pull/4455/commits/d56f0fcf997515fb5c2ea5556e9c3edcc9edda86